_posts/2022-04-25-tardigrade-launch.md

```diff
@@ -0,0 +1,301 @@
+---
+layout: post
+title:  "ETL in Trino speeds with project Tardigrade: now ready for early users"
+author: Andrii Rosa, Brian Zhan, Lukasz Osipiuk, Zebing Lin
+excerpt_separator: <!--more-->
+---
+
+After 6 months of challenging work, we’ve launched Project Tardigrade. It is 
+designed to improve user experience of running resource intensive queries that
+are common in ETL space. It’s available right now! It required some significant
+and fascinating engineering to get us here. Read on to learn how Project 
```

Comment:
to get us to the current status. The latest Trino release includes all the work from Project Tardigrade. Read on to learn how it all works and how to enable the fault tolerant query processing in Trino.